eng/targets/Settings.props

```diff
@@ -20,7 +20,7 @@
 
     <Features>strict</Features>
     <ProduceReferenceAssembly>true</ProduceReferenceAssembly>
-    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <GenerateDocumentationFile Condition="'$(GenerateDocumentationFile)' == ''">true</GenerateDocumentationFile>
```

Comment:
Strictly speaking I think this change is unnecessary. The build in this case is setting `/p:GenerateDocumentationFile=true` which means it's a global property. Those are essentially immutable. Even explicitly setting it to `false` here won't change anything.

Reply:
This observation is true. Reverting.